### DISCUSSION TITLE
ERROR: initializing node: starting node: could not build arguments for function...

### DISCUSSION THREAD

**Discussion by chenhonghe** [filecoin-project/lotus#7822] [Build and Sync]
lotus version 1.5.2+mainnet+git.09e78a0fe

ERROR: initializing node: starting node: could not build arguments for function "github.com/filecoin-project/lotus/node/modules/lp2p".StartListening.func1 (/home/pool/lotus/node/modules/lp2p/addrs.go:98): failed to build host.Host: could not build arguments for function "reflect".makeFuncStub (/usr/local/go/src/reflect/asm_amd64.s:12): failed to build lp2p.BaseIpfsRouting: could not build arguments for function "reflect".makeFuncStub (/usr/local/go/src/reflect/asm_amd64.s:12): failed to build dtypes.NetworkName: could not build arguments for function "reflect".makeFuncStub (/usr/local/go/src/reflect/asm_amd64.s:12): failed to build *store.ChainStore: could not build arguments for function "reflect".makeFuncStub (/usr/local/go/src/reflect/asm_amd64.s:12): failed to build dtypes.ChainBlockstore: could not build arguments for function "reflect".makeFuncStub (/usr/local/go/src/reflect/asm_amd64.s:12): failed to build dtypes.UniversalBlockstore: received non-nil error from function "reflect".makeFuncStub (/usr/local/go/src/reflect/asm_amd64.s:12): failed to open badger blockstore: file does not exist for table 728018

How do I start lotus???

**Comment by rcarfora** [comment]
having this issue as well. did you get a fix?

**Comment by Reiers** [comment]
Hi @chenhonghe 

Thanks for the report. The issue you are experiencing lies elsewhere.

Moving this issue to Lotus Discussions for help and troubleshooting.

Closing Lotus issue ticket.

**Comment by LexLuthr** [comment]
@chenhonghe Do you still need help with this problem in the latest version of the lotus?

**Reply by LexLuthr** [reply]
I am archiving this discussion as there has been no new input from the author for more than 7 days.
